packages/widgets/src/components/NotificationCard/types.ts

```diff
@@ -0,0 +1,12 @@
+import { IMapIcons } from '@ssa-ui-kit/core';
+
+export interface INotificationCardProps {
+  title: string | JSX.Element;
+  text: string | JSX.Element;
+  isRead: boolean;
+  badgeColor: string;
```

Comment:
Is this the correct type for the color? Shouldn't we use `keyof MainColors` as the `<Badge />` does?

Reply:
done